### PR TITLE
db/adjust-to-product-name

### PR DESCRIPTION
**Pull request by karolkolodziej**
- [x] Main directory name
- [x] Demo directory (waiting for the `highchars-utils` RP to be merged)
- [x] Test demo
- [x] Class
- [x] File names
- [x] References in code 
- [x] Demos
- [x] Modules imports
- [x] Docs
- [x] builders
- [x] webpack
- [x] Licence
- [x] Cypress
- [x] CSS
- [x] class name

### CODE REVIEW

**Comment by sebastianbochan** [comment]
162 files, nice perspective for review :D

**Review by sebastianbochan — APPROVED** [review]

**Review by sebastianbochan — APPROVED** [review]

File: tools/webpacks/dashboard.webpack.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const ResolveTypeScriptPlugin = require("resolve-typescript-plugin").default;
```
Comment (bre1470) [inline tools/webpacks/dashboard.webpack.js:6]: The name of the webpack configuration itself should become plural as well.

**Review by bre1470 — APPROVED** [review]
Nice! Looks good. 👍 

One last thing...

**Comment by bre1470** [comment]
@pawelfus What do we do with the branch names?

**Comment by pawelfus** [comment]
> @pawelfus What will we do with the branch names?

Dev branches: up to you, `db/*` imho is great
Main: `dashboard` -> `dashboards`, but it's not part of this task (I will take care of it)
MVP: `dashboards/mvp` nothing changes

**Pull request merged**